src/pages/myNotes/componentInfo.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import NoteListItem from "./noteListItem";
+import { StyledComponentInfo } from "./styled";
+
+const ComponentInfo = ({ id, description, title, date }) => {
+  return (
+    <StyledComponentInfo className="active">
+      {!id ? (
+        <h1>Select note to display</h1>
+      ) : (
+        <NoteListItem
+          id={id ? id : null}
```

Comment:
remove the condition, 'id' should always exist

Reply:
Done